tests/Fixture/ArrayAccessIteratorFixture.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace IterTools\Tests\Fixture;
+
+class ArrayAccessIteratorFixture implements \ArrayAccess, \Iterator
```

Comment:
Question: Is this class necessary? Could the tests just use the built-in `\ArrayIterator` as that also implements `\ArrayAccess` and `\Iterator`? Or is there something else this fixture iterator provides that the built-in iterator does not?

Reply:
Done: caa0b57d48d9484fdc1fc77c5a497a164ae45d9a.